PED/SetPedPhonePaletteIdx.md

```diff
@@ -0,0 +1,127 @@
+---
+ns: PED
+---
+## SET_PED_PHONE_PALETTE_IDX
+
+```c
+// 0x83A169EABCDB10A2
+void SET_PED_PHONE_PALETTE_IDX(Ped ped, int index);
+```
+
+Sets the palette index of a ped's phone.
+
+| Value | Color   |
+| :---: | :-----: |
+|  `0`  | Blue    |
```

Comment:
Actually `0` is light Blue theme from the settings menu.

Reply:
👍